ci/kind/test-e2e-kind.sh

```diff
@@ -142,6 +142,7 @@ function setup_cluster {
 function run_test {
   TMP_DIR=$(mktemp -d $(dirname $0)/tmp.XXXXXXXX)
   curl -o $TMP_DIR/antrea.yml https://raw.githubusercontent.com/antrea-io/antrea/main/build/yamls/antrea.yml
+  sed -i -e "s|image: \"projects.registry.vmware.com/antrea/antrea-ubuntu:latest\"|image: \"antrea/antrea-ubuntu:latest\"|g" $TMP_DIR/antrea.yml
```

Comment:
@heanlan do you think it might make sense to move the change of image registry (commit da2b2cd437f2c8e3f6a089834add8739da67258d) into a standalone PR and quickly merge it?

Reply:
of course! the PR is opened here: https://github.com/antrea-io/theia/pull/38